user/registration/build.gradle

```diff
@@ -14,4 +16,7 @@ android {
 
 dependencies {
     api project(Modules.core)
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
```

Comment:
Check if we can add these dependencies in deps.kt

Reply:
comments are removed in latest commit